Readme.md

```diff
@@ -546,6 +547,33 @@ next((Test Class, 13))
 completed
 ```
 
+### toggleBool
+
+The `toggleBool(initial:)` operator returns an observable sequence of booleans, starting with `initial` and toggling its value.
+
+```swift
+let input = [1, 2, 3, 4]
+    
+print("Input:", input)
+print("Output:")
+    
+Observable.from(input)
+    .toggleBool(initial: false)
+    .subscribe(onNext: { result in
+        print(result)
+    })
+```
+
+```
+Input: [1, 2, 3, 4]
```

Comment:
Can you align the output with other operators, e.g. just print 

next(false)
next(true)
next(false)
next(true)
next(false)
completed

Reply:
Done